package.json

```diff
@@ -111,14 +111,17 @@
     ]
   },
   "scripts": {
-    "vscode:prepublish": "tsc -p ./",
-    "compile": "tsc -watch -p ./",
-    "preinstall": "node ./scripts/preinstall.js",
+    "vscode:prepublish": "npm run compile",
+    "compile": "tsc -p ./",
+    "watch": "tsc -watch -p ./",
+    "preinstall": "npm i --save mvn-artifact-download && node ./scripts/preinstall.js",
```

Comment:
isn't to avoid having to call "npm install --ignore-scripts"?

Reply:
I suppose yes, I added this because if we wanted to execute npm install it will always failed on missing mvn dependency and I did not found how to solved that other way.

Reply:
it sounds cool.
May I ask you to create a different PR just for that to separate added functionalities please?
it will ease review to have them separately.
Also please update:
- the travis.yml file and remove the` npm install --ignore-scripts`
- the contributing.md and remove the  npm install --ignore-scripts`

Reply:
sure, I will do that :slightly_smiling_face:

Reply:
Created #140